src/src/WebServer/ToolsPage.cpp

```diff
@@ -150,7 +150,7 @@ void handle_tools() {
         }
       } else {
         html_B(F("WARNING"));
-        addHtml(F(" OTA not possible."));
+        addHtml(F("Not enough space to safely update. Update might fail. "));
```

Comment:
You're missing a space here, now it will look like '**WARNING**Not ...'

Reply:
Thanks, didn't notice that 👍